tests/ops/api/v1/endpoints/test_system.py

```diff
@@ -273,6 +274,26 @@ def test_patch_connection_secrets_removes_access_token_for_client_config(
         assert resp.status_code == HTTP_200_OK
         assert resp.json()["items"][0]["authorized"] is False
 
+    def test_system_patch_hidden(
```

Comment:
it'd be nice for this test to assert that the updates are actually reflected in the db! there have been many times where an endpoint gives a `200` and the correct response body but it doesn't actually make the intended change to the db :)

Reply:
assert added here: https://github.com/ethyca/fides/pull/5600/commits/61f38a4fb0fb9c6884690d4b4f61b03a6fb37e90#diff-5e73ddc03e8d539e57124c63d6225fdffa0412ad12eafbfede16ac016efd8e8eR297